static/elements/chromedash-feature-page.js

```diff
@@ -225,7 +225,7 @@ export class ChromedashFeaturePage extends LitElement {
               </a>
             </span>
           `: nothing}
-          ${this.user && this.user.can_edit ? html`
+          ${this.user && this.user.can_edit_all ? html`
```

Comment:
Don't you also need to check if this feature's ID is in the  list of IDs that the signed in user can edit?

Reply:
Yes that's true! I this this is a WIP page created by Kevin, so the tests don't run against this page. I'll add the functionality regardless so that it works with the new changes.